src/redux/hooks/appHooks.ts

```diff
@@ -0,0 +1,8 @@
+import { useSelector } from 'react-redux';
+
+import { RootState } from '@redux/store';
+
+// eslint-disable-next-line
+export function useAppSelector<T extends (_state: RootState) => any>(selector: T): ReturnType<T> {
+  return useSelector(selector);
+}
```

Comment:
NB: would be nice to have `useAppDispatch` too, instead of `useDispatch<AppThunkDispatch>();`

Reply:
I am using the redux-thunk middleware for redux, so it needs the AppDispatchThunk type
https://redux.js.org/usage/usage-with-typescript#define-typed-hooks